tests/integration/features/source_code_traceability/_source_nodes/40_remap/strictdoc_config.py

```diff
@@ -0,0 +1,27 @@
+from strictdoc.core.project_config import ProjectConfig
+
+
+def create_config() -> ProjectConfig:
+    config = ProjectConfig(
+        project_title="StrictDoc Documentation",
+        project_features=[
+            "REQUIREMENT_TO_SOURCE_TRACEABILITY",
+            "SOURCE_FILE_LANGUAGE_PARSERS",
+        ],
+        source_nodes = [
+            {
+                "path": "src/",
```

Comment:
I feel like it is a good time to adapt this dictionary of key-values to be a properly typed class. How about we make source nodes to be a `list[SourceNodesEntry]` so that the parsed ProjectConfig directly results in a list of SourceNodesEntry without an intermediate conversion from dict?